### PR TITLE
CASSANDRA-17266 Remove default_time_to_live for Views in describe

### PR DESCRIPTION
**Pull request by yashLadha**
Remove default_time_to_live in the describe output for views as it is
not supported by them. Updated to pass the isView variable to the
CqlBuilder.

Fixes: https://issues.apache.org/jira/browse/CASSANDRA-17266

### CODE REVIEW

**Review by blerer — COMMENTED** [review]
Thanks the patch looks good :-) . Could you just add a materialized view to the `testDescribeRoundtrip()` to validate that we do not hit another problem when we try to recreate the view?

**Comment by yashLadha** [comment]
@blerer Updated the `testDescribeRoundtrip()` test with materialized view creation and assertion.

**Review by blerer — APPROVED** [review]

File: src/java/org/apache/cassandra/schema/TableParams.java [hunk 1]
```diff
@@ -273,26 +273,31 @@ public void appendCqlTo(CqlBuilder builder)
                .append("AND compression = ").append(compression.asMap())
                .newLine()
                .append("AND crc_check_chance = ").append(crcCheckChance)
-               .newLine()
+               .newLine();
+
+        if (!isView) {
```
Comment (driftx) [inline src/java/org/apache/cassandra/schema/TableParams.java:278]: Brace should be on newline.

**Review by driftx — APPROVED** [review]
+1 w/nit

File: src/java/org/apache/cassandra/schema/TableParams.java (hunk 1 above)
Comment (blerer) [reply src/java/org/apache/cassandra/schema/TableParams.java:278]: I can take care of that on commit :-)

**Pull request closed without merging**